app/views/loans/addloancollateral.html

```diff
@@ -23,10 +23,11 @@ <h4>{{ 'label.heading.addloancollateral' | translate }}</h4>
                     </div>
                 </div>
                 <div class="form-group">
-                    <label class="control-label col-sm-2" for="value">{{ 'label.input.value' | translate }}</label>
+                    <label class="control-label col-sm-2" for="value">{{ 'label.input.value' | translate }}<span
+                            class="required">*</span></label></label>
 
                     <div class="col-sm-3">
-                        <input id="value" type="text" class="form-control" ng-model="formData.value">
+                        <input id="value" type="text" class="form-control" ng-model="formData.value" required>
```

Comment:
@MYKatz Thanks, Good job. Please also add a `late-validate` attribute and a `form-validate` tag. we want to keep the UI consistent :). Hint: Refer to this PR: https://github.com/openMF/community-app/pull/2271/files